docs/hub/spaces-changelog.md

```diff
@@ -4,6 +4,25 @@ title: Spaces Changelog
 
 <h1>Spaces Changelog</h1>
 
+## [2022-05-21] - Default Debian pakcages
+- Spaces environment now comes with pre-installed popular packages (`ffmpeg`, `libsndfile1`, etc.)
```

Comment:
should we document the list of installed packages more exhaustively?

Reply:
Not sure it worths it.
The idea is that it will just work out-of-the-box.
If it ever doesn't, then the user has to be advanced enough to fix the issue (using container logs and / or build logs will to debug) anyway and I'm not sure that having a list in the documentation would help